### PR TITLE
docs: correct controller base path

### PR DESCRIPTION
**Pull request by br4mch**
File paths contain the 'models' directory. Replaced by 'controllers'.

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by achrinza** [comment]
Hi @br4mch, thanks for the contribution! Could you complete the steps as per-[Submitting a pull request](https://loopback.io/doc/en/lb4/submitting_a_pr.html)?

tldr;
[ ] Sign the [CLA](https://cla.strongloop.com/agreements/strongloop/loopback-next)
[ ] Use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
[ ] [Rebase and squash](https://loopback.io/doc/en/lb4/submitting_a_pr.html#6-rebasing) into 1 commit

**Comment by dhmlau** [comment]
@achrinza, you're a minute ahead of me. :)

@br4mch, since your changes are about fixing the file path in the code sample, could you please also improve the commit message to match the purpose of this PR? Thanks!

**Comment by br4mch** [comment]
Yes, I'm sorry... Not that experienced with the whole GitHub flow yet 😅. I tried my best in following the steps. Only issue I'm having now is that `npm test` isn't passing because of a lint issue in the corrected file. Since I correctly changed the `model` part of the path to `controllers`, I have no clue what I'm looking for... Maybe someone else has an idea?

**Comment by achrinza** [comment]
@br4mch No worries! There seems to be 2 issues:

1 . Commit message
  The lb4 repo follows a certain commit message structure; Conventional Commits (Angular 2+). See my previous comment regarding this
2. Linting
  This can be fixed with `npm run lint:fix`

Be sure to rebase and squash after adding the new commit.

**Comment by br4mch** [comment]
I believe all steps were executed correctly

**Comment by dhmlau** [comment]
@br4mch, thanks for your contribution. Your PR has landed! 🎉
